runtime/lua/vim/diagnostic.lua

```diff
@@ -228,6 +229,14 @@ end
 --- See |nvim_buf_set_extmark()|.
 --- @field virt_text_hide? boolean
 
+--- @class vim.diagnostic.Opts.VirtualLines
+---
+--- Only show diagnostics for the current line.
+--- @field only_current_line? boolean
+---
+--- The characters used to draw the lines between diagnostic messages.
+--- @field drawing_characters? {cross?:string, horizontal?:string, horizontal_up?:string, up_right?:string, vertical?:string, vertical_right?:string}
```

Comment:
Wonder if instead of indicating the characters here we want to expose options like "rounded"/"rectangular" etc

Reply:
I'm not sure we need to make this configurable at all, at least for now. I think we hard code it for now and if people ask for it, we can make it an option. But if possible, the fewer knobs the better.

Reply:
Agreed that we can skip it for the initial version.

For future versions, we can consider using a format similar the `border` parameter for `nvim_open_win`.